smart-contract/hyperledger-fabric/v2/java/src/main/java/hu/bme/mit/ftsrg/tpcc/registry/EntityRegistry.java

```diff
@@ -0,0 +1,113 @@
+package hu.bme.mit.ftsrg.tpcc.registry;
+
+import hu.bme.mit.ftsrg.tpcc.entities.EntityFactory;
+import hu.bme.mit.ftsrg.tpcc.entities.SerializableEntityInterface;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import org.hyperledger.fabric.contract.Context;
+import org.hyperledger.fabric.shim.ledger.CompositeKey;
+import org.hyperledger.fabric.shim.ledger.KeyValue;
+
+public class EntityRegistry implements RegistryInterface {
+  // private static final Logger LOGGER = Logger.getLogger(EntityRegistry.class.getName());
+
+  Context ctx;
```

Comment:
You do not need this, all methods get a `Context` as a parameter and you cannot even factor that out to the class-level.